askup/utils/general.py

```diff
@@ -34,6 +35,9 @@
     ) as ranked
     {}
 '''
+PROFILE_RANK_USERS_COUNT = '''
```

Comment:
Why multiple docstring? Usual string is seems to be fair enough.

Reply:
You right. This construction is left from the first version of this feature. Now it's needed no more.